### PR TITLE
Adjust roslyn mcs dll deploy list based on #13755

### PR DESCRIPTION
**Pull request by kg**

### CODE REVIEW

**Comment by kg** [comment]
@monojenkins build makeinstall

**Comment by kg** [comment]
@monojenkins build Linux x64 - prefix sanity checks

File: mcs/packages/Makefile [hunk 1]
```diff
@@ -19,11 +19,8 @@ ROSLYN_FILES_FOR_MONO = \
 	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.dll		\
 	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.Scripting.dll \
 	$(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll	\
-	$(ROSLYN_CSC_DIR)/System.Numerics.Vectors.dll		\
 	$(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll	\
 	$(ROSLYN_CSC_DIR)/System.Runtime.CompilerServices.Unsafe.dll	\
-	$(ROSLYN_CSC_DIR)/System.Threading.Tasks.Extensions.dll	\
```
Comment (marek-safar) [inline mcs/packages/Makefile]: These two are mobile profiles only and we'll have to copy them from somewhere

Comment (kg) [reply mcs/packages/Makefile]: So we do want to include them in the roslyn repo and copy them in? Or only copy them for mobile profiles? I'm not clear on what this means

Comment (marek-safar) [reply mcs/packages/Makefile]: We'll have to copy them from roslyn

Comment (kg) [reply mcs/packages/Makefile]: OK. Should we do this for all profiles?

Comment (marek-safar) [reply mcs/packages/Makefile]: yes (we install roslyn for net_4_x only)

**Comment by kg** [comment]
@monojenkins build failed

**Pull request merged**